### DISCUSSION TITLE
About Microsoft Dolby Digital Atmos Pack

### DISCUSSION THREAD

**Discussion by neptune84x** [SimonMacer/AnWave#21] [Open Discussion]
First of all, I would like to thank you for this wonderful work.
My question is that I want to extract only the Microsoft Dolby Digital Atmos Pack from anwave. 
Is this possible? Or can you offer the Microsoft Dolby Digital Atmos Pack as a separate package?

**Comment by SimonMacer** [comment]
I released Microsoft Dolby Digital Atmos Pack standalone installer. Now. You can download standalone installer to install.

【HOW-TO】Restore Microsoft Dolby Digital Decoder/Encoder MFT support in Windows 11, version 24H2:https://github.com/SimonMacer/AnWave/discussions/14

Microsoft Dolby Digital Atmos Pack version 10.0.22621.4169 standalone installer Download:
[AnWave 2024 Split Feature Release Download](https://github.com/SimonMacer/AnWave/releases/tag/AnWave-Split)

**Reply by JourneyOver** [reply]
Is there a way to run the install silently? would like to add it to my install scripts that I use whenever I do a clean install of windows but from the looks of things it currently can't be used in a silent manner :/

**Comment by anupdebnath** [comment]
Hey there! I hope you're doing well. I was just wondering if you received a DMCA about this? I can't seem to find any of the releases available for download. Thanks!

**Reply by SimonMacer** [reply]
> Hey there! I hope you're doing well. I was just wondering if you received a DMCA about this? I can't seem to find any of the releases available for download. Thanks!

Hi.

The old Microsoft Dolby Digital Atmos Pack is deprecated. Now, you can re-enable the Dolby AC-3 FoD package in AnWave for Windows 11 24H2.

https://github.com/SimonMacer/AnWave?tab=readme-ov-file#restore-microsoft-dolby-digital-decoderencoder-mft-and-wordpad-support-in-windows-11-version-24h2
